### DISCUSSION TITLE
Is there documentation for navigation sections?

### DISCUSSION THREAD

**Discussion by FireLemons** [squidfunk/mkdocs-material#4794] [General]
What determines the order items appear in sections?

I see you can do
```
---
status: new
---
```

at the top of a md file to show a new badge next to it. Can you set status to anything else?

**Comment by squidfunk** [comment]
The feature you mention is called [navigation statuses](https://squidfunk.github.io/mkdocs-material/reference/#setting-the-page-status) – it's currently solely used to render the icons in the navigation, but we might expand that in the future. No order of pages is associated with it.
